src/App.jsx

```diff
@@ -1,13 +1,28 @@
 import './App.css'
-import logo from './assets/logo-white.png'
+import {Route, Routes} from "react-router-dom";
+import Home from './pages/home/Home.jsx'
+import NewPost from './pages/newpost/NewPost.jsx'
+import Overview from './pages/overview/Overview.jsx'
+import Error from './pages/error/Error.jsx'
+import Navigation from "./components/navigation/Navigation.jsx";
+import PostDetail from "./pages/postdetail/PostDetail.jsx";
+
+
 
 function App() {
     return (
-        <div className="page-container">
-            <img src={logo} alt="Company logo"/>
-            <h1>Begin hier met het maken van jouw blog-applicatie!</h1>
-        </div>
-    )
+        <>
+            <Navigation/>
+            <Routes>
+                <Route path="/" element={<Home/>}/>
+                <Route path="/new-post" element={<NewPost/>}/>
+                <Route path="/overview" element={<Overview/>}/>
+                <Route path="/error" element={<Error/>}/>
```

Comment:
<Route path="*" element={<Error/>} /> met je huidige path zal de error page nooit aangeroepen worden. met path="*" wel. 
```suggestion
                <Route path="/error" element={<Error/>}/>
```